src/queries.ts

```diff
@@ -15,6 +16,26 @@ namespace Queries {
     return nodes;
   };
 
+  export function getReturnType(functionNode: n.FunctionDeclaration): string {
+    let result;
+
+    //handle type reference as array for example
+    if (get(functionNode, 'returnType.typeAnnotation.type') === 'TSTypeReference') {
```

Comment:
You can create a constans.ts or strings.json file in which to put read-only(const) variable

Reply:
This string currently is used in one place. Ergo is no reason to put this in constant as long as there is no repetition.